aea/cli/packages.py

```diff
@@ -0,0 +1,200 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""Package manager."""
+
+
+import json
+from collections import OrderedDict
+from pathlib import Path
+from typing import Optional
+from typing import OrderedDict as OrderedDictType
+
+import click
+
+from aea.cli.utils.click_utils import component_flag
+from aea.cli.utils.context import Context
+from aea.cli.utils.decorators import pass_ctx
+from aea.configurations.data_types import PackageId, PublicId
+from aea.helpers.dependency_tree import DependencyTree
+from aea.helpers.io import open_file
+from aea.helpers.ipfs.base import IPFSHashOnly
+
+
+try:
+    from aea_cli_ipfs.registry import fetch_ipfs  # type: ignore
+
+    IS_IPFS_PLUGIN_INSTALLED = True
+except (ImportError, ModuleNotFoundError):
+    IS_IPFS_PLUGIN_INSTALLED = False
+
+PACKAGES_FILE = "packages.json"
+
+
+@click.group("packages")
+@click.pass_context
+def package_manager(
+    click_context: click.Context,  # pylint: disable=unused-argument
+) -> None:
+    """Local package manager."""
+
+
+@package_manager.command()
+@pass_ctx
+def sync(ctx: Context) -> None:
+    """Sync local packages."""
+
+    if not IS_IPFS_PLUGIN_INSTALLED:
+        raise click.ClickException(
+            "Please install ipfs plugin using `pip3 install open-aea-cli-ipfs`"
+        )
```

Comment:
This is just for the demo purpose, I'm working on adding support for fetching from multiple registries